### PR TITLE
#158857770 Implement designs for order page

### PR DESCRIPTION
**Pull request by Dannytebj**
### What does this PR do?
This PR implements the design for the order page mock to meet design expectations

### What are the relevant pivotal tracker stories?
[158857770](https://www.pivotaltracker.com/story/show/158857770)

### Any background context you want to add?
None

### Screenshot
screen shot 2018-07-12 at 7 54 18 pm

### CODE REVIEW

**Comment by Charpell** [comment]
Looks good to me

File: src/components/Order/Orders.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <div className="wrapper">
+        <Navbar />
+        <Sidenav />
+        <div className="orders-wrapper">
+          <h3>Make Orders</h3>
+
+          <div className="orders-container">
+            <div className="date-wrapper">
+              <h3>
+                JUNE 2018
```
Comment (ibrahim013) [inline src/components/Order/Orders.jsx]: @daniel is there any reason why this is hardcoded, what happens when the month is no more `June 2018`

File: src/components/Order/Orders.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    this.state = {};
+  }
+
+  /**
+   *
+   *
+   * @returns {void}
+   * @memberof Orders
+   */
+  renderDates() {
```
Comment (ibrahim013) [inline src/components/Order/Orders.jsx]: @daniel I also don't seem to see any form of state management in your code base, no action dispatch and no reducers this shows that you are not implementing the redux flow which believes that the `store` is the only source of truth, is there any reason why this is so?

File: src/index.js [hunk 3]
```diff
@@ -12,12 +12,11 @@ const store = configureStore();
 
 
 // eslint-disable-next-line
-Raven.config(config.SENTRY_URL)
-  .install();
+// Raven.config(config.SENTRY_URL).install();
```
Comment (ibrahim013) [inline src/index.js]: @daniel, is this meant to be commented out?

File: src/tests/components/Login.test.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import Login from '../../pages/Login/Login.jsx';
+import Login from '../../components/Login/Login';
```
Comment (ibrahim013) [inline src/tests/components/Login.test.js]: @daniel your commit also show that this component isn't tested, this does not align with our `TDD` standard

File: src/components/Order/Orders.jsx (hunk 2 above)
Comment (Dannytebj) [reply src/components/Order/Orders.jsx]: @ibrahim013 As you can see, the title of this PR is: `implement designs for orders page`. the keyword there is `designs`. There is another task to hook it up with redux. Same goes for your first question.

**Pull request merged**